package.json

```diff
@@ -16,7 +16,6 @@
     "@types/babel__generator": "^7.0.0",
     "@types/babel__template": "^7.0.0",
     "@types/dedent": "0.7.0",
-    "@types/jest": "24.0.2",
```

Comment:
this is intentionally lower to make sure we don't pull in a version with dependencies - it causes "race" conditions when compiling since `@types/jest` tries to load the types of `jest-diff` which haven't been built yet.

Allowing the `*` one to stay at 26 _might_ work, not sure

Reply:
we might be able to fix it by having some sort of `types` exclusion? not sure

Reply:
I've just realised that the `MockedFunction` type I wanted to show is actually in `24.9.0`. So I've added that back in and removed the 26.